config/printer-creality-ender3-s1-2021.cfg

```diff
@@ -0,0 +1,128 @@
+# This file contains pin mappings for the stock 2021 Creality Ender 3
+# S1. To use this config, during "make menuconfig" select the
+# STM32F103 with a "28KiB bootloader" and serial (on USART1 PA10/PA9)
+# communication.
+
+# If you prefer a direct serial connection, in "make menuconfig"
+# select "Enable extra low-level configuration options" and select
+# serial (on USART3 PB11/PB10), which is broken out on the 10 pin IDC
+# cable used for the LCD module as follows:
+# 3: Tx, 4: Rx, 9: GND, 10: VCC
+
+# Flash this firmware by copying "out/klipper.bin" to a SD card and
+# turning on the printer with the card inserted. The firmware
+# filename must end in ".bin" and must not match the last filename
+# that was flashed.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[stepper_x]
+step_pin: PC2
+dir_pin: PB9
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA5
+position_endstop: 0
+position_max: 220
+position_min: -5
+homing_speed: 50
+
+[stepper_y]
+step_pin: PB8
+dir_pin: PB7
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA6
+position_endstop: 0
+position_max: 220
+position_min: -5
+homing_speed: 50
+
+[stepper_z]
+step_pin: PB6
+dir_pin: !PB5
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 8
+endstop_pin: probe:z_virtual_endstop
+position_max: 270
+position_min: -3
+
+[extruder]
+step_pin: PB4
+dir_pin: PB3
+enable_pin: !PC3
+microsteps: 16
+gear_ratio: 3.5:1
+rotation_distance: 20
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PA1
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PC5
+control: pid
+pid_Kp: 22.865
+pid_Ki: 1.292
+pid_Kd: 101.178
+min_temp: 0
+max_temp: 250
```

Comment:
for full metal sprite extruder, the max temp is 300

Reply:
I know, thats why there is a Ender 3 S1 Pro config, look over stuff before immediately prompting a review.